models/categories.js

```diff
@@ -0,0 +1,8 @@
+const db = require('../db/connection');
+
+exports.selectCategories = () => {
+    return db.query('SELECT * FROM categories;')
+    .then((result) => { return result.rows})
```

Comment:
If you are comfortable doing so, we could deconstruct rows in the .then parameter then simply return rows. Just a comment not essential.